pkgs/applications/networking/misc/zammad/default.nix

```
@@ -129,6 +136,7 @@ stdenv.mkDerivation {
   passthru = {
     inherit rubyEnv yarnEnv;
     updateScript = [ "${callPackage ./update.nix {}}/bin/update.sh" pname (toString ./.) ];
+    passthru.tests.zammad = nixosTests.zammad;
```

```suggestion
		tests = { inherit (nixosTests) zammad; };
```

passthru.passthru....